cargo-shuttle/src/lib.rs

```diff
@@ -398,31 +398,34 @@ impl Shuttle {
         let path = if needs_path {
             let path = args
                 .path
-                .to_str()
-                .context("path arg should always be set")?;
+                .join(project_args.name.as_ref().expect("name should be set"));
```

Comment:
Could we keep the context, rather than expect? Or does it not provide a lot of value?

Reply:
expect makes sense since the error would be due to flawed logic in the code before this